python_examples/intervention_example/case_isolation_simulation_flow.py

```diff
@@ -0,0 +1,231 @@
+sim = pe.routine.Simulation()
+sim.configure(
+    population,
+    [pe.sweep.InitialInfectedSweep(), pe.sweep.InitialisePlaceSweep()],
+    [
+        pe.sweep.InterventionSweep(),
+        pe.sweep.UpdatePlaceSweep(),
+        pe.sweep.HouseholdSweep(),
+        pe.sweep.PlaceSweep(),
+        pe.sweep.SpatialSweep(),
+        pe.sweep.QueueSweep(),
+        pe.sweep.HostProgressionSweep(),
+    ],
+    sim_params,
+    file_params,
+)
+sim.run_sweeps()
+
+# Need to close the writer object at the end of each simulation.
+del sim.writer
+del sim
+
+###############
+# 100% isolates
+name_parameter_file = 'case_isolation_parameters_100isolate.json'
+name_output_file = 'output_100isolate.csv'
+
+# Set config file for Parameters
+pe.Parameters.set_file(os.path.join(os.path.dirname(__file__),
+                       name_parameter_file))
+
+# Method to set the seed at the start of the simulation, for reproducibility
+pe.routine.Simulation.set_random_seed(seed=30)
+
+# Create population
+population = pe.routine.FilePopulationFactory.make_pop(file_loc,
+                                                       random_seed=42)
+
+# Configure population with input data
+pe.routine.ToyPopulationFactory.add_places(population, 1)
+
+# file_params give details for where output should be written to.
+file_params = {"output_file": name_output_file,
+               "output_dir": os.path.join(os.path.dirname(__file__),
+                                          "intervention_outputs"),
+               "spatial_output": True}
+
+# Create a simulation object, configure it with the parameters given, then
+# run the simulation.
+sim = pe.routine.Simulation()
+sim.configure(
+    population,
+    [pe.sweep.InitialInfectedSweep(), pe.sweep.InitialisePlaceSweep()],
+    [
+        pe.sweep.InterventionSweep(),
+        pe.sweep.UpdatePlaceSweep(),
+        pe.sweep.HouseholdSweep(),
+        pe.sweep.PlaceSweep(),
+        pe.sweep.SpatialSweep(),
+        pe.sweep.QueueSweep(),
+        pe.sweep.HostProgressionSweep(),
+    ],
+    sim_params,
+    file_params,
+)
+sim.run_sweeps()
+
+# Need to close the writer object at the end of each simulation.
+del sim.writer
+del sim
+
+###############################
+# Creation of a plot of results
+logging.getLogger("matplotlib").setLevel(logging.WARNING)
+filename_0isolate = os.path.join(os.path.dirname(__file__),
+                                 "intervention_outputs",
+                                 "output_0isolate.csv")
+filename_50isolate = os.path.join(os.path.dirname(__file__),
+                                  "intervention_outputs",
+                                  "output_50isolate.csv")
+filename_100isolate = os.path.join(os.path.dirname(__file__),
+                                   "intervention_outputs",
+                                   "output_100isolate.csv")
+
+df_0isolate = pd.read_csv(filename_0isolate)
+df_50isolate = pd.read_csv(filename_50isolate)
+df_100isolate = pd.read_csv(filename_100isolate)
```

Comment:
Small thing but you could generate these in a loop, iterating over a list [0, 50, 100] - this makes it much easier to adapt the script when you want to try different parameters. Come to think of it, you could write the main body of the script in the same way

Reply:
Good point! I now put the main body and the plotting in a loop. Could you check the code @KCGallagher ?

Reply:
Yep that loop looks good @HenrietteCapel! Feel free to re-request a review on this PR once you've done all the previous changes (I see there are still a few comments from the old review outstanding at the moment)